configs/cascade_rcnn/cascade-mask-rcnn_r50_fpn_ms-3x_coco.py

```diff
@@ -1,4 +1,4 @@
 _base_ = [
-    '../common/mstrain_3x_coco_instance.py',
-    '../_base_/models/cascade_mask_rcnn_r50_fpn.py'
+    '../common/ms_3x_coco_instance.py',
```

Comment:
![image](https://user-images.githubusercontent.com/26483343/186651496-1cd7ae39-d5a7-4290-afba-5c478f697cfb.png)

why `ms_3x_coco_instance.py`? Should be `ms-3x_coco-instance`?